src/functionsShellCommandAction.ts

```diff
@@ -61,11 +83,11 @@ module.exports = async function (options) {
         process.exit();
       }
 
-      var initializeContext = function (context) {
-        _.forEach(emulator.triggers, function (trigger) {
+      const initializeContext = (context: any) => {
+        _.forEach(emulator.triggers, (trigger) => {
```

Comment:
Optional - it would be pretty easy to refactor this chunk to use plain JS instead of lodash, and then we're one step closer to removing the dependency for good

Reply:
I got rid of `_.forEach` and `_.includes` but `_.set` with dot-separated path notation is a bit tricky so I left that. We'll probably need to create our own `utils.set` to replace that throughout the codebase and that seems out of scope for this PR.